projects/mci/src/mci_output.c

```diff
@@ -59,6 +60,8 @@ static void prv_handle_drive(SoftTimerId timer_id, void *context) {
   MotorCanDriveCommand drive_command = { 0 };
   EEDriveOutput drive_state = drive_fsm_get_drive_state();
   bool is_cruise = drive_fsm_is_cruise();
+  uint8_t is_regen_brake = get_regen_braking_state();
```

Comment:
Can we make this a `bool`? SImilar to `is_cruise`? Or is there something preventing us from doing this?

Reply:
I used the enum typedef for it which `RegenBrakingState`